src/update-feeds/dapi-data-registry.ts

```diff
@@ -45,7 +76,7 @@ export const decodeReadDapiWithIndexResponse = (
       heartbeatInterval,
     },
     dataFeedValue: { value, timestamp },
-    dataFeed,
+    dataFeed: decodedDataFeed,
```

Comment:
I think we could do a `decodedDataFeed: decodedDataFeed` to make this more explicit.